staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go

```diff
@@ -491,7 +489,6 @@ func makeManagedFieldsUpdater(parser *gvkParser, convert convertor, manager stri
 			return nil, fmt.Errorf("failed to convert old object to unstructured: %v", err)
 		}
 
-		newObjUnstructured := unstructured.Unstructured{Object: newObjMap}
```

Comment:
A lot of the code before that can also be removed if you used the `TypeConverter` directly.

Actually, I've hidden the `gvkParser` because it's only used when you're trying to convert from `runtime.Object` to `typed.Value`, so everything is under that interface.